openslides_backend/datastore/reader/flask_frontend/flask_frontend.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
```

Comment:
Why do we need the reader application and the whole flask frontend?

Reply:
It is used (at least) in the migration tests. The same goes for the writer. They can theoretically be removed when the adjusted reader/writer are implemented, but then the tests have to be adjusted as well. For now, it's easiest to keep them around.